### PR TITLE
Journey AI Fatigue and Score schemas

### PR DESCRIPTION
**Pull request by josemathew**
PR to add 2 new schemas and profile extension in campaign namespace. Issue #491

### CODE REVIEW

File: extensions/adobe/experience/campaign/journeyaifatigue.schema.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/journeyaifatigue",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey AI Fatigue",
```
Comment (lrosenthol) [inline extensions/adobe/experience/campaign/journeyaifatigue.schema.json]: while this is part of an Adobe extension - is the naming here too product and use specific?

File: extensions/adobe/experience/campaign/journeyaiscores.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "description": "Journey AI scores for one channel",
+  "definitions": {
+    "journeyaiscores": {
+      "properties": {
+        "xdm:fatigue": {
+          "title": "Fatigue",
+          "description": "Fatigue score for the profile",
+          "$ref": "https://ns.adobe.com/experience/campaign/journeyaifatigue"
+        },
+        "xdm:openScoresBySendHourForWeek": {
```
Comment (lrosenthol) [inline extensions/adobe/experience/campaign/journeyaiscores.json]: Will customers understand this, because I do not.

Comment (josemathew) [reply extensions/adobe/experience/campaign/journeyaiscores.json]: We just wanted to add some context about the scores ( that they for a week, 24x7 scores ). I think title and description can be used for that, and I will rename the fields as `openScores` and `clickScores`

File: extensions/adobe/experience/campaign/journeyaifatigue.schema.json (hunk 1 above)
Comment (josemathew) [reply extensions/adobe/experience/campaign/journeyaifatigue.schema.json]: I will change the title to be less specific about Journey AI. Should I have to change the id ( `journeyaifatigue` ) as well?

**Comment by cmathis** [comment]
@josemathew  - Are you wanting these new schemas referenced by a new ExpereienceEvent campaign extension field - if so you will need to add a new field to the /extension/experience/campaign/experienceevent.schema.json file as well.

**Comment by josemathew** [comment]
@cmathis I will checking in a Profle schema extension today.

**Comment by lrosenthol** [comment]
Looks good to me.  I am OK with merging it.  @kstreeter , you?

**Comment by cfraser** [comment]
@kstreeter please include this PR in 0.9.6.

File: extensions/adobe/experience/campaign/profile.schema.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/campaign/profile",
+  "title": "Adobe Campaign Profile Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Adobe Campaign extension for Profile",
+  "definitions": {
+    "profile": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/journeyai/email": {
```
[the remark this answers is not in the document]
Comment (josemathew) [reply extensions/adobe/experience/campaign/profile.schema.json]: @kstreeter I will create a sub-object and add email field into it.

**Comment by kstreeter** [comment]
Hi @josemathew, can you take a look at #515, @cdegroot-adobe had a few additional comments on the PR and has proposed some small changes.  Let us know if you have any concerns!

**Pull request merged**